storage/local/interface.go

```diff
@@ -91,6 +94,17 @@ type Preloader interface {
 		fp model.Fingerprint,
 		from model.Time, through model.Time,
 	) (SeriesIterator, error)
+	PreloadInstant(
+		fp model.Fingerprint,
+		timestamp model.Time, stalenessDelta time.Duration,
+	) (SeriesIterator, error)
 	// Close unpins any previously requested series data from memory.
 	Close()
 }
+
+// ZeroSamplePair is the pseudo zero-value of model.SamplePair used by the local
+// package to signal a non-existing sample. It is a SamplePair with timestamp
+// model.Earliest and value 0.0. Note that the natural zero value of SamplePair
```

Comment:
You mean the part about the "natural zero value"? I find that quite valuable to answer the FAQ that will inevitably show up for anybody who looks at this variable.